website/docs/d/l3_ext_subnet.html.markdown

```diff
@@ -22,16 +31,21 @@ data "aci_l3_ext_subnet" "example" {
 
 ```
 
-## Argument Reference
+## Argument Reference ##
 
-- `external_network_instance_profile_dn` - (Required) Distinguished name of parent External Network Instance Profile object.
-- `ip` - (Required) IP address of Object l3 extension subnet.
+* `external_network_instance_profile_dn` - (Required) Distinguished name of the parent External Network Instance Profile object.
+* `ip` - (Required) IP address of the L3 Extension Subnet object.
 
-## Attribute Reference
+## Attribute Reference ##
 
-- `id` - Attribute id set to the Dn of the l3 extension subnet.
-- `aggregate` - (Optional) Aggregate Routes for l3 extension subnet.
-- `annotation` - (Optional) Annotation for object l3 extension subnet.
-- `description` - (Optional) Description for object l3 extension subnet.
-- `name_alias` - (Optional) Name alias for object l3 extension subnet.
-- `scope` - (Optional) The list of domain applicable to the capability.
+* `id` - Attribute id set to the Dn of the L3 Extension Subnet object.
+* `aggregate` - (Optional) Aggregate Routes of the L3 Extension Subnet object.
+* `annotation` - (Optional) Annotation of the L3 Extension Subnet object.
+* `description` - (Optional) Description of the L3 Extension Subnet object.
+* `name_alias` - (Optional) Name alias of the L3 Extension Subnet object.
+* `scope` - (Optional) The list of domain applicable to the capability.
+* `relation_l3ext_rs_subnet_to_profile` - (Optional) Relation to Route Control Profile (class rtctrlProfile). Type: Block.
+	* `tn_rtctrl_profile_name` - **Deprecated** (Optional) Associates the External EPGs with the Route Control Profiles.
+	* `tn_rtctrl_profile_dn` - (Optional) Associates the External EPGs with the Route Control Profiles.
+	* `direction` - (Optional) Relation to configure route map for each BGP peer in the inbound and outbound directions.
+* `relation_l3ext_rs_subnet_to_rt_summ` - (Optional) Relation to class rtsumARtSummPol.
```

Comment:
Relation to a Route Summarization Policy (class rtsumARtSummPol)

Reply:
ok